### PR TITLE
[Semi-Modular] GAGS Teshari items

### PR DESCRIPTION
**Pull request by Ebin-Halcyon**
## About The Pull Request

This PR adds GAGSified items, like basic jumpsuits, skirts, shoes, ~~poncho~~ Being worked on later, sneakers, and berets that fit on Teshari snugly.

Images of said sprites:
Beret + shoes + jumpskirt
image

Jumpsuit + shoes
image

## How This Contributes To The Skyrat Roleplay Experience

Tesharti sprites :)
Also makes adding Teshari clothing for GAGS easier since things are being converted to it left and right.

## Changelog

:cl:
imageadd: Select few GAGSified Teshari items
/:cl:

### CODE REVIEW

**Comment by Tastyfish** [comment]
FYI you can make `greyscale_config_worn_teshari` work by going to `code\game\objects\items.dm` and adding the following inside the Skyrat changed section of `/obj/item/update_greyscale()`:

```dm
if(greyscale_config_worn_teshari)
  worn_icon_teshari = SSgreyscale.GetColoredIconByType(greyscale_config_worn_teshari, greyscale_colors)
```

Disclaimer: I'm just freehanding in the comment box this but I think that'll work.

File: modular_skyrat/modules/GAGS/greyscale_configs.dm [hunk 1]
```diff
@@ -42,6 +42,18 @@
 	name = "Beret With Fancy Badge Worn - Vox"
 	icon_file = 'modular_skyrat/modules/GAGS/icons/beret_vox.dmi'
 
+/datum/greyscale_config/beret_badge_fancy/worn/teshari
```
Comment (ghost) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm:53]: just a note, try to sort this by type, not by species; basically how the vox does it:
item
item/worn
item/worn/vox
(then you'd have item/worn/tesh)

File: modular_skyrat/modules/GAGS/greyscale_configs.dm [hunk 2]
```diff
@@ -106,6 +118,11 @@
 	name = "Ranger Poncho Worn"
 	json_config = 'modular_skyrat/modules/GAGS/json_configs/ranger_poncho/ranger_poncho_worn.json'
 
+/datum/greyscale_config/ranger_poncho/teshari
```
Comment (ghost) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm]: make sure this is a subtype of ranger_poncho/worn

Comment (ghost) [reply modular_skyrat/modules/GAGS/greyscale_configs.dm]: you also wont need a new json_config if you use the same icon states btw, only the icon_file

**Comment by Ebin-Halcyon** [comment]
Guhg, i'll fix it tomorrow i'm tired as fuck.

**Comment by Ebin-Halcyon** [comment]
Finally.

**Comment by Ebin-Halcyon** [comment]
>Suddenly failing linters and map build

agony.

**Comment by hugmeorelse** [comment]
Won't be needing this because of #12480

**Comment by Ebin-Halcyon** [comment]
Sobbing.

**Comment by Ebin-Halcyon** [comment]
hey it passes :)

**Review by Gandalf2k15 — APPROVED** [review]
fine, but know this will probably break with kapu limbs

**Comment by Ebin-Halcyon** [comment]
**Woe.**

**Pull request merged**